lib/openshift/console_notification.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class ConsoleNotification < ClusterResource
+    RESOURCE = "consolenotifications"
```

Comment:
we need to use full api name like `lib/openshift/config_imageregistry_operator_openshift_io.rb`

Reply:
@akostadinov If update to:
```
class ConsolenotificationConsoleOpenshiftIo < ClusterResource
    RESOURCE = "consolenotifications.console.openshift.io"
```
It will raise undefined step for the step "Admin ensure... is deleted after scenario"

Reply:
You must have a typo somewhere. dots should not make a difference for that step.

Reply:
@akostadinov Dot does matter for the string match inside step definition, updated, plz check, thanks!
Btw recent steps always failed on Job but could pass locally. (refer to https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/46634/console)